packages/medusa/src/api/routes/admin/orders/index.ts

```diff
@@ -21,6 +21,9 @@ export default (app, featureFlagRouter: FlagRouter) => {
   if (featureFlagRouter.isFeatureEnabled(SalesChannelFeatureFlag.key)) {
     relations.push("sales_channel")
   }
+  if (featureFlagRouter.isFeatureEnabled(OrderEditingFeatureFlag.key)) {
```

Comment:
thought: maybe we should remove this as a default relation altogether - think it is fair to have the consumer call the edits endpoint separately instead of creating a huge order object.